wdl2cwl/main.py

```diff
@@ -71,43 +71,48 @@ def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
         """Load WDL workflow and convert to CWL."""
         inputs: List[cwl.WorkflowInputParameter] = []
         outputs: List[cwl.WorkflowOutputParameter] = []
+        seen: Set[str] = set()
         wf_steps: List[cwl.WorkflowStep] = []
         wf_name = obj.name
-        wf_description = obj.meta["description"]
+        wf_description = obj.meta["description"] if "description" in obj.meta else None
         for call in obj.body:
-            call_name = call.name  # type: ignore
             callee = call.callee  # type: ignore
-            namespace, _ = call.callee_id  # type: ignore
+            local_call_name = call.name  # type: ignore
+            callee_id = f"{call.callee_id[0]}.{local_call_name}" if len(call.callee_id) == 2 else local_call_name  # type: ignore
```

Comment:
The local name is the name for the task within the space. like ` call Bowtie2 as Forward` in this case the name of the task is Bowtie2 while the name of the step is Foward. This is not the same as the callee id.